tutorials/modules.html

```diff
@@ -0,0 +1,646 @@
+      is avaiable just below - it shows sample countries and cities and although it looks small it will generate over 345 KB (Gzipped):
+    </p>
+      <img src="images/modularization_comparison.png" alt="modularization_comparison_graph" />
+    <p>
+     You can compare the following examples to see the difference in KB of the final build.
+     Note: this is an example in a nutshell, the next section shows how to do it step by step.
+    </p>
+     <pre><code class="lang-js">
+// import a full build, all functionalities included
+import Handsontable from 'handsontable',
+
+const container = document.createElement('div');
+document.body.appendChild(container);
+const hot = new Handsontable(container, {
+  colHeaders: true,
+  filters: true,
+  data: [
+    {
+      city: "Fontainebleau",
+      country: "France",
+    },
+    {
+      city: "Milton",
+      country: "United Kingdom",
+    },
+    {
+      city: "Giedlarowa",
+      country: "Poland",
+    },
+    {
+      city: "Forssa",
+      country: "Finland",
+    },
+    {
+      city: "Halle",
+      country: "Germany",
+    }
+  ],
+  dropdownMenu: true,
+  columns: [
+    { data: 'city' },
+    {
+      data: 'country',
+      type: 'dropdown',
+      source: ['Finland', 'France', 'Germany', 'Poland', 'United Kingdom'],
+    },
+  ],
+  filters: true,
+});
+      </code></pre>
+      <p>
+      Thanks to using modules you can achieve exactly same result but it will generate less KB.
+      These steps decreased the final bundle to 200 KB (Gzipped).
+      </p>
+      <pre><code class="lang-js">
+// import the base only
+import Handsontable from 'handsontable/base';
+// choose cell types you want to use and import them
+import { registerCellType, DropdownCellType } from 'handsontable/cellTypes';
+// choose plugins you want to use and import them
+import {
+  registerPlugin,
+  AutoColumnSize,
+  CopyPaste,
+  Filters,
+} from 'handsontable/plugins';
+
+// register imported cell types and plugins
+registerCellType(DropdownCellType);
+registerPlugin(AutoColumnSize);
+registerPlugin(CopyPaste);
+registerPlugin(Filters);
+
+// the rest of code remain exactly same as in the previous example!
+const container = document.createElement('div');
+document.body.appendChild(container);
+
+const hot = new Handsontable(container, {
+  colHeaders: true,
+  filters: true,
+  data: [
+    {
+      city: "Fontainebleau",
+      country: "France",
+    },
+    {
+      city: "Milton",
+      country: "United Kingdom",
+    },
+    {
+      city: "Giedlarowa",
+      country: "Poland",
+    },
+    {
+      city: "Forssa",
+      country: "Finland",
+    },
+    {
+      city: "Halle",
+      country: "Germany",
+    }
+  ],
+  dropdownMenu: true,
+  columns: [
+    { data: 'city' },
+    {
+      data: 'country',
+      type: 'dropdown',
+      source: ['Finland', 'France', 'Germany', 'Poland', 'United Kingdom'],
+    },
+  ]
+});
```

Comment:
I have an idea, we could extract the settings from both examples.
```suggestion
const hot = new Handsontable(container, settings);
```